### PR TITLE
Added functionality to show who a player has voted for.

### LINKED ISSUE #181: Better Role Interactions
**What are the tasks that need to be done? Please put your name inside the brackets to assign yourself**
- [x] (Daniel) Show Medics who they are saving
- [x] (Tom) Show Detectives if the person they checked is good or bad
- [x] (Daniel) Show Mafia who they are killing
- [x] (Daniel) Show Players who they are voting for

**Note: This is currently blocked by #187. Once it has been merged, you should be able to complete these tasks.**

### In-Depth Feature Description
Role interactions are quite poor at the moment. These enhancements will improve the game UX. 
Could use something like an outline around a selected player?

For Detectives:
Use the state from GameContext to get checkedPlayers and then show revealed players

### Images for Reference:
None, sorry!

### Feature Approval:
- [x]  Feature approved by: @Dhors

### PR DESCRIPTION
**Pull request by Dhors**
# Description
This PR fixes #181

Added ability to show client who they have voted for. Both for night abilities and for day time trial vote.
Changed styling to colour the highlighted player

# Demo
image

# Type of change:
* New feature (Non-breaking change that adds functionality)

### CODE REVIEW

File: frontend/src/Components/Player.jsx [hunk 1]
```diff
@@ -38,12 +40,15 @@ export default function Player({ playerId, playerName, style, childRef }) {
...
                 });
+                dispatch( { type:'show-selected' , status: `Selected ${playerName} for ability`, votedPlayer: playerName } );
                 break;
             case 'discussion':
                 socket.emit(`day-vote`, { votingFor: playerName });
+                dispatch( { type:'show-selected' , status: `Voted ${playerName} for trial`, votedPlayer: playerName } );
                 break;
             case 'trial':
                 socket.emit(`trial-vote`, { votingFor: playerName });
+                dispatch( { type:'show-selected' , status: `Voted to hang ${playerName}`, votedPlayer: playerName } );
```
Comment (reeved) [inline frontend/src/Components/Player.jsx:51]: Could we please change 'hang' to 'kill'? :)

**Review by reeved — APPROVED** [review]
LGTM! just a suggestion about some text :)

File: frontend/src/Components/Player.jsx (hunk 1 above)
Comment (Dhors) [reply frontend/src/Components/Player.jsx]: done

**Review by ateagit — APPROVED** [review]
LGTM

**Review by reeved — COMMENTED** [review]
Gucci

**Pull request merged**